### ISSUE TITLE
Implement experiment script

### ISSUE THREAD

**Issue by bveeramani** [bveeramani/eyecite-ml#17]
Write a script take takes the following arguments:
* `--batch` (int): the batch size.
* `--epochs` (int): the number of epochs to train over.
* `--optimizer` (string): the name of the optimizer class.
* `--lr` (float): the learning rate.
* `--dataset` (string): the name of the data set class.
* `--model` (string): the name of the model class.
* '--height` (int): the height of input images.
* `--width` (int): the width of input images.

The script should generate a folder populated with the following items:
* `parameters.csv`: A CSV file that contains the date, CUDA version, PyTorch version, Python version, operating system, metadata version (this can just be 1.0.0 for now), repository commit hash, and all of the arguments listed above.  
* `loss.csv`: A CSV file containing the loss at each batch of every epoch.
* `confusion.csv`: A CSV file containing the confusion matrix.
* `scores.csv`: A CSV file containing the average IOU score, F1-score, log loss, and pixel accuracy.
* `example.md`: A markdown file showing the loss scores, segmentation, and image of a single sample.
* `model.pt`: The model that was just trained. Can be named something other than `model.pt`.

The folder should be uploaded to Box.

**Comment by bveeramani** [comment]
Resolved by commit `9577f6c`.
